game-core/src/main/java/games/strategy/engine/framework/startup/WatcherThreadMessaging.java

```diff
@@ -12,14 +12,31 @@
 
 /** Interface to communicate lobby watcher events to user. */
 public interface WatcherThreadMessaging {
-  void serverNotAvailableHandler(String message);
+  String COMPUTER_NOT_REACHABLE_ERROR_MESSAGE =
+      "Your computer is not reachable from the internet.\n"
+          + "Please make sure your Firewall allows incoming connections (hosting) "
+          + "for TripleA.\n"
+          + "(The firewall exception must be updated every time a new version of "
+          + "TripleA comes out.)\n"
+          + "And that your Router is configured to send TCP traffic the correct port "
+          + " to your local ip address.\n"
+          + "See 'How To Host...' in the help menu, at the top of the lobby "
```

Comment:
What is the "How to host" button currently doing?
We might want to mention the possibility of using a relay service in the future

Reply:
Good question, this is something to update. We no longer have a 'how to host' menu option. The reason it was removed is that the game was linking to many forum posts, some/many of which were getting out of date. Instead of having that type of situation, instead all help content should be linked to the user guide and then we maintain just one link to the user guide.

Reply:
If the 'how to host' page is updated to mention a relay service, then we'll have that done.